.github/workflows/release.yml

```
@@ -0,0 +1,424 @@
+        BUILD_TIME: ${{ github.event.head_commit.timestamp }}
+        GIT_COMMIT: ${{ github.sha }}
+      run: |
+        echo "Building for $GOOS/$GOARCH..."
+        
+        # Build with release flags
+        go build \
+          -trimpath \
+          -ldflags="-s -w -X main.Version=v$VERSION -X main.BuildTime=$BUILD_TIME -X main.GitCommit=$GIT_COMMIT" \
+          -o goingenv-$GOOS-$GOARCH \
+          ./cmd/goingenv
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)